models/devicereport.go

```diff
@@ -18,6 +18,7 @@ import (
 	"encoding/json"
 )
 
+// DeviceReport has been deprecated and needs to be removed.
```

Comment:
It looks like there is some style associated with deprecation, namely, having a line starting with `"Deprecated: "` and explaining what to use instead or what the user should be trying to accomplish instead of using this.

[This blog post has some links to Go standard library docs making use of the convention with discussion on the `golang` repo](https://rakyll.org/deprecated/)

[Here's the official wiki on the matter.](https://github.com/golang/go/wiki/Deprecated)

Reply:
Reformatted content using guidelines above.